server/logger/logger.go

```diff
@@ -0,0 +1,18 @@
+package logger
+
+import (
+	"log"
+
+	"go.uber.org/zap"
+)
+
+// Logger zap logger instance
+var Logger *zap.Logger
+
+func init() {
+	l, err := zap.NewProduction()
```

Comment:
I have not used zap so correct me if I am wrong, don't we need to initialize it with config level? Or don't we have configurable log levels like debug/warn/etc?
Also, any specific reason we are creating a package level logger object?

Reply:
Ya just wanted basic logger. Later will integrate with config object to control log levels.

Reply:
We will move to the salt logger soon.